src/main/kotlin/com/tealcube/minecraft/bukkit/mythicdrops/items/MythicCustomItem.kt

```diff
@@ -120,7 +121,8 @@ internal data class MythicCustomItem(
                 repairCost = configurationSection.getInt("repair-cost", DEFAULT_REPAIR_COST),
                 isEnchantmentsRemovableByGrindstone = isEnchantmentsRemovableByGrindstone,
                 isAddDefaultAttributes = isAddDefaultAttributes,
-                hdbId = configurationSection.getNonNullString("hdb-id")
+                hdbId = configurationSection.getNonNullString("hdb-id"),
+                rgb = configurationSection.getNonNullString("rgb")
```

Comment:
Continuing from the other comments:
```suggestion
                rgb = Rgb(
                  red = configurationSection.getInt("rgb.red"),
                  green = configurationSection.getInt("rgb.blue"),
                  blue = configurationSection.getInt("rgb.green")
                )
```